### DISCUSSION TITLE
상품 판매 상세페이지 유스케이스 정의서

### DISCUSSION THREAD

**Discussion by nanazzoo** [FFarmFFarm/FarmFarm#5] [유스케이스 정의서]
>
## 1. 개요
액터(사용자)가 상품 상세페이지를 조회할 수 있는 유스케이스이다.
<br/>

>
## 2. 관련 액터
비회원, 구매자, 판매자, 관리자
<br/>
>
## 3. 사전 조건
없음.
<br/>
>
## 4. 사후 조건
상품 페이지 조회 후 구매 또는 문의하기, 후기 등의 기능은 로그인을 요함
<br/>

>
## 5. 이벤트 흐름

<br/>

>
### 5-1. 정상 흐름(로그인한 회원일 때)

<br/>

>
#### 액터가 판매 상품 목록에서 판매 상품을 클릭할 시 
- [ ] 해당 상품 상세페이지로 이동

<br/>

#### 상세 페이지 조회 시 
- [ ] 상품 이미지, 이름, 가격, 단위, 지역, 결제 방식, 상품 상세 설명, 후기 목록 조회 가능

<br/>

#### 상품 문의하기 버튼 클릭 시 
- [ ] 판매자와의 채팅방으로 이동

<br/>

#### 상품 구매하기 버튼 클릭 시 
- [ ] 단위와 수량 지정 후 결제창 열림
- [ ] 결제 완료 시 마이페이지 주문 목록으로 이동

<br/>

#### 후기 목록 조회 시
- [ ] 상품 구매자가 남긴 후기 확인 가능

<br/>

### 5-2. 예외 흐름(로그인한 회원일 때)

<br/>

#### 구매하기 버튼 클릭 시(팜팜페이 등록 안 한 경우)
- [ ] 팜팜페이 등록 화면으로 이동

<br/>

### 5-3. 예외 흐름(비회원일 때)

<br/>

#### 구매하기 버튼 클릭 시
- [ ] 로그인 페이지로 이동
 
<br/>

#### 문의하기 버튼 클릭 시
- [ ] 로그인 페이지로 이동

### 5-4. 예외 흐름(판매자 회원일 때)

- [ ] 구매하기 버튼 안 보임
- [ ] 문의하기 버튼 안 보임

**Comment by luejenie** [comment]
양식이 너무 좋네요 퍼가요~♡
